docs/operators/move.md

```diff
@@ -0,0 +1,49 @@
+## `move` operator
+
+The `move` operator moves (or renames) a field from one location to another.
+
+It's configured by passing 'to' and 'from' fields.
+
+### Configuration Fields
+
+| Field      | Default          | Description                                                                                                                                                                                                                              |
+| ---        | ---              | ---                                                                                                                                                                                                                                      |
+| `id`       | `restructure`    | A unique identifier for the operator                                                                                                                                                                                                     |
+| `output`   | Next in pipeline | The connected operator(s) that will receive all outbound entries                                                                                                                                                                         |
+| `from`      | required       | The field to move the value out of.   
+| `to`      | required       | The field to move the value into.
```

Comment:
Can we link to the doc for `field`, so people can easily find out what syntax is expected here?